src/toHaveStyleRule.js

```diff
@@ -20,10 +20,18 @@ const getClassNames = received => {
 const hasClassNames = (classNames, selectors) =>
   classNames.some(className => selectors.includes(`.${className}`))
 
-const getRules = (ast, classNames) =>
-  ast.stylesheet.rules.filter(
+const getRules = (ast, classNames, options) => {
+  let rules = ast.stylesheet.rules
+  if (options.media) {
+    rules = rules
```

Comment:
Oh, you are right - I missed that.
I think we can avoid setting a default value, just `(received, property, value, options)` would work 👍